actionview/CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
-*   `number_to_currency` and `number_with_delimiter` now accept custom `delimiter_pattern` option 
-     to handle placement of delimiter, to support currency formats like INR 
-     
-     Example: 
-        
+*   Add a `hidden_field` on the `collection_radio_buttons` to avoid raise a error
```

Comment:
There is something strange in this change… it should only show your new lines, not replace the existing one.
Did you accidentally edit the next CHANGELOG message too?

If you review this, also fix `avoid raise` to `avoid raising` :wink: